tests/integration/test_integration.py

```diff
@@ -0,0 +1,199 @@
+import time
+
+from flask import Flask
+
+
+def test_main(app: Flask):
```

Comment:
Perhaps split this test ?

Reply:
perhaps later when this becomes really bulky. it will be cut into separate methods, but for now it's ok. It is to be a big integration test with dependent steps.